common/pom.xml

```diff
@@ -22,5 +22,56 @@
             <groupId>org.jsoup</groupId>
             <artifactId>jsoup</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-webmvc</artifactId>
+            <version>4.3.14.RELEASE</version>
```

Comment:
@Simsteve7 Versions should come from the parent pom's <dependency-management>. Same for dependencies below